PagingWithNetworkSample/app/src/test-common/java/com/android/example/paging/pagingwithnetwork/repository/FakeRedditApi.kt

```diff
@@ -53,41 +51,45 @@ class FakeRedditApi : RedditApi {
     private fun findSubReddit(subreddit: String) =
             model.getOrDefault(subreddit, SubReddit())
 
-    override fun getTop(subreddit: String, limit: Int): Call<RedditApi.ListingResponse> {
+    override suspend fun getTop(subreddit: String, limit: Int): RedditApi.ListingResponse {
```

Comment:
this still has three methods - does this compile?

Reply:
Done